packages/atomic/src/components/recommendations/atomic-recs-result/atomic-recs-result.tsx

```diff
@@ -161,8 +161,9 @@ export class AtomicRecsResult {
     }
     return (
       // deepcode ignore ReactSetInnerHtml: This is not React code
-      <Host class={resultComponentClass}>
+      <Host exportparts="result-list-grid-content" class={resultComponentClass}>
```

Comment:
I needed this exports for more styling of the content inside the grid.

Reply:
I didn't know you could add `exportparts` on the root of a component, TIL.

If you want to re-export parts though, I would prefer if it was explicitly from each parent component, so that we know we need to document them with `@part`.